requirements.txt

```diff
@@ -97,7 +97,6 @@ WebTest==2.0.24
 Werkzeug==0.11.15
 WTForms==2.1
 xvfbwrapper==0.2.9
--e .
```

Comment:
dh-virtualenv was having issues with modules installed from source and this seemed like a quicker fix. Let me see if I can work around it...